docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -0,0 +1,69 @@
+// tag::ess[]
+
+// QUESTION: Should we add a warning here about updating to 7.13 and include a
+// link to the upgrade docs? 
+
+{ecloud} runs a hosted version of {fleet-server}. To use {fleet-server}, start
+the service:
+
+//TODO: Describe how to start the service.
+
+When you're done, log in to {kib} and go to *Management > {fleet}*. Click the
+*Agents* tab to confirm that {fleet-server} is running and healthy.
+
+//TODO: Add screen capture.
+
+On the *Policies* tab, notice that the {agent} is enrolled in the Elastic Cloud
+policy. This policy is managed by {ecloud}. You cannot modify it.
+
+// end::ess[]
+
+// tag::self-managed[]
+
+To deploy {fleet-server} on a self-managed cluster, install an {agent} and
+enroll it in the default {fleet-server} policy. The policy is created
+automatically when you run {fleet} for the first time.
+
+To add {fleet-server}:
+
+//QUESTION: Do we want to show how to call the API to generate the token, or
+//walk users through the easy setup with the UI?
```

Comment:
This likely shouldn't need API docs, the token generation is exposed in the UI now... a last minute addition that will help users greatly.  see screenshot:
![Screen Shot 2021-04-26 at 6 38 35 PM](https://user-images.githubusercontent.com/12970373/116159686-ac523d80-a6be-11eb-8200-40f5c425510d.png)

Reply:
I think we might want to have two different sections: this section, which points users to the UI, and a separate section that describes how to do this from a command line. I tried to do a hybrid with the Elastic Agent steps for 7.12 and that was a little confusing to users. I think we might need two sections: the "happy path" through the UI, and detailed steps (command, API calls for generating token, etc) for folks who are using automation or want to do everything from one CLI. WDYT?